Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM segment/chamber:2.3 AS chamber
+FROM golang:1.13 as builder
+WORKDIR /go/src/github.com/buildkite/buildkite-agent-scaler
+COPY . .
+RUN GO111MODULE=off GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o buildkite-agent-scaler .
+
+FROM alpine:3.9
```

Comment:
Do we use segment-cached versions of alpine and golang containers? Probably not, just asking.

Reply:
Good call. We do have a scratch image that should work. I updated.